### DISCUSSION TITLE
Unable to reapply for github student developer pack since it says Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by KaushikPrasad** [community/community#76886] [GitHub Education]
### Hi everyone,

Hello Github, I'm having issues with passing the verification for github student developer pack. I have a school email and also uploaded the required proof of my school id and inspite of that it got rejected. I am unable to reapply since it says  Your uploaded proof has already been used previously.

**Comment by dakshithadissanayaka** [comment]
It seems there might be a glitch or issue with your verification process for the GitHub Student Developer Pack. Since you've already uploaded the required proof of your school ID and received the "Your uploaded proof has already been used previously" message, it suggests that the system is recognizing the document but unable to properly process it or link it to your account.

To resolve this issue, you can try the following steps:

1. **Contact GitHub Support:** Since the issue persists despite providing the required proof, it's best to reach out to GitHub support directly. They can investigate the issue further and provide more specific troubleshooting steps or identify any underlying issues with the verification process. Explain the situation clearly, including the previous rejection and the "Your uploaded proof has already been used previously" message.

2. **Provide Additional Documentation:** If GitHub support suggests you provide additional documentation, consider providing a transcript or enrollment document along with your school ID. This might help verify your student status without relying solely on the school ID.

3. **Check for Updates:** Keep an eye out for any updates or announcements from GitHub regarding the verification process. They might be aware of the issue and working on a fix.

4. **Wait for a Fix:** If the issue persists and GitHub support is unable to resolve it immediately, consider waiting for a while and retrying the verification process. The system might be updated or fixed in the meantime.

Remember to be patient and persistent in communicating with GitHub support. They will be able to assist you effectively in resolving the verification issue and granting you access to the GitHub Student Developer Pack.

**Comment by partiid** [comment]
same problem here, very frustrating..
